src/dmd/builtin.d

```diff
@@ -306,6 +430,56 @@ public extern (C++) void builtin_init()
     add_builtin("_D3std4math5ldexpFNaNbNiNfdiZd", &eval_ldexp);
     add_builtin("_D3std4math5ldexpFNaNbNiNffiZf", &eval_ldexp);
 
+    add_builtin("_D3std4math3logFNaNbNiNfeZe", &eval_log);
+    add_builtin("_D3std4math3logFNaNbNiNfeZe", &eval_log);
+    add_builtin("_D3std4math3logFNaNbNiNfeZe", &eval_log);
```

Comment:
You are adding the same function 3 times here. If the float and double overloads don't exist one builtin should be enough. Same for a number of other functions.